src/components/site/SummaryList.js

```diff
@@ -0,0 +1,111 @@
+  const iconCssClassesHours = classnames("side-list-ico", iconClock);
+  const iconCssClassesEmail = classnames("side-list-ico", iconEnvelop);
+  const iconCssClassesPhone = classnames("side-list-ico", iconMobile);
+  const iconCssClassesBaltcogo = classnames("side-list-ico", iconCircle);
+  const iconCssClassesDirector = classnames("side-list-ico", iconUser);
+  return (
+    <React.Fragment>
+      <div className="dg_summary-list">
+        <div className="col-md-12 bg-light px-4 py-4">
+          <div className="row mb-3">
+            <div className="col-2">
+              <i className={iconCssClassesMain}></i>
+            </div>
+            <div className="col-10">
+              <h3>Main Office:</h3>
+              <p>
+                111 West Chesapeake Avenue
+                <br /> Suite 307
+                <br /> Towson, Maryland 21204
+              </p>
+            </div>
+          </div>
+
+          <div className="row mb-3">
+            <div className="col-2">
+              <i className={iconCssClassesHours}></i>
+            </div>
+            <div className="col-10">
+              <h3>Hours:</h3>
+              <p>
+                Monday through Friday
+                <br />8 a.m. to 4:30 p.m.
+              </p>
+            </div>
+          </div>
+
+          <div className="row mb-3">
+            <div className="col-2">
+              <i className={iconCssClassesEmail}></i>
+            </div>
+            <div className="col-10">
+              <h3>Email:</h3>
+              <p>
+                <a href="#">publicworks@​baltimorecountymd.gov</a>
+              </p>
+            </div>
+          </div>
+
+          <div className="row mb-3">
+            <div className="col-2">
+              <i className={iconCssClassesPhone}></i>
+            </div>
+            <div className="col-10">
+              <h3>Phone:</h3>
+              <p>410-887-6789</p>
+              <h3>Emergency Dispatch:</h3>
+              <p>410-887-6789</p>
+            </div>
+          </div>
+          <div className="row mb-3">
+            <div className="col-2">
+              <i className={iconCssClassesBaltcogo}></i>
+            </div>
+
+            <div className="col-10">
+              <h3>BALTCOGO311:</h3>
+              <p>
+                <a href="#">Report a problem</a> online.
+              </p>
+            </div>
+          </div>
+          <div className="row mb-3">
+            <div className="col-2">
+              <i className={iconCssClassesDirector}></i>
+            </div>
+
+            <div className="col-10">
+              <h3>Director:</h3>
+              <p>Steve Walsh, P.E.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </React.Fragment>
+  );
+};
+
+ SummaryList.propTypes = {
+
+  /** Font Awesome icon class (eg: iconMarker, iconClock, iconEnvelop, iconMobile, iconCircle, iconUser*/
+  iconMarker: PropTypes.string.isRequired,
+  /** label for that row */
```

Comment:
please remove any commented code that is not being used.

Reply:
done